watchmal/dataset/h5_dataset.py

```diff
@@ -117,14 +117,9 @@ def __getitem__(self, item):
         start = self.event_hits_index[item]
         stop = self.event_hits_index[item + 1]
 
-        hit_pmts = self.hit_pmt[start:stop].astype(np.int16)
-        hit_charges = self.hit_charge[start:stop]
-        hit_times = self.time[start:stop]
-
-        data = {"hit_pmts" : hit_pmts,
-                "hit_charges" : hit_charges,
-                "hit_times" : hit_times}
-        data_dict["data"] = data
```

Comment:
This seems to be the main propagated change. I can't think of any problems this would cause, and removing the data addition step here actually makes more sense (not going to accidentally pass the wrong data in another class)

Reply:
Yeah, saving these things to the class member data instead of passing through as elements of the dictionary made more sense I think, and just made things a lot simpler for more complicated class structures.